src/Main.java

```diff
@@ -1,5 +1,69 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Home work 11");
+        task1();
+        task2();
+        task3();
+
     }
-}
\ No newline at end of file
+
+    public static void checkForLeapYear(int year1) {
+        boolean leapYear = year1 > 1584 && year1 % 4 == 0 || year1 % 400 == 0;
```

Comment:
```suggestion
        boolean leapYear = year1 > 1584 && year1 % 4 == 0 && year1 % 100 != 0 || year1 % 400 == 0;
```
вот такое условие должно быть